src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
```

Comment:
Consider using uppercase to name constants 
<img width="356" alt="Screenshot 2024-02-16 at 14 20 48" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/135232001/968b7b77-540f-4771-b253-a628f62cfaa2">